src/proc.py

```diff
@@ -29,16 +29,27 @@
 from subprocess import * # flake8: noqa
 
 
-def FixPython(cmd):
+def Which(filename, cwd):
+  to_search = ([cwd] + os.environ['PATH'].split(os.pathsep))
+  for path in to_search:
+    abs_path = os.path.join(path, filename)
+    if os.path.exists(abs_path):
```

Comment:
Should use the same logic as `IsExe` from https://github.com/WebAssembly/waterfall/pull/143/commits/5d9cc2e32e2242c6c5c08c6051e4f6b7c1e20cd3